index.html

```diff
@@ -40,9 +40,12 @@ <h1 class="site-header__title">
         <a href="/" class="site-header__text">astoria.digital</a><span class="site-header__cursor">_</span>
       </h1>
       <p class="site-header__slogan">Digital volunteers for the community</p>
-      <a href="mailto:team@astoria.digital?subject=feedback" class="site-header__contact">
+      
+      <!-- <a href="mailto:team@astoria.digital?subject=feedback" class="site-header__contact">
```

Comment:
Is this link no longer valid? If it's not should it be completely removed instead of commented out?

Reply:
It's a styling idea, instead of having that link in the header I was thinking of including it later in the page